### ISSUE TITLE
macOS keyboard shortcut documentation note

### ISSUE THREAD

**Issue by johnblommers** [vkbo/novelWriter#124]
With one exception noted below, of the documented keyboard shortcuts that use the <kbd>Ctrl</kbd> key, Macintosh users should replace <kbd>Ctrl</kbd> with <kbd>Command</kbd>.

Exception: <kbd>Command</kdb><kbd>h</kbd> does not open the search and display bar because it is intercepted by the system to hide the foremost app, so pressing these two keys hides novelWriter.

**Comment by vkbo** [comment]
Not a mac user, so don't know. Are you saying that macOS replaces they key shortcut? So, it's only a matter of making the note in the manual?

**Comment by johnblommers** [comment]
macOS has always used the Command key <kbd>⌘</kbd> instead of the <kbd>Ctrl</kbd> key for common keyboard shortcuts. So  <kbd>⌘</kbd> <kbd>c</kbd> is copy, <kbd>⌘</kbd> <kbd>v</kbd> is paste and <kbd>⌘</kbd> <kbd>x</kbd> is cut.

<kbd>⌘</kbd> <kbd>h</kbd> hides the current app.

On macOS, when you view a menu, its shortcut is typically displayed next to it. For novelWriter all of the keyboard shortcuts display with a <kbd>⌘</kbd> symbol instead of a <kbd>Ctrl</kbd> symbol.

**Comment by vkbo** [comment]
Yeah, that makes sense as I use the Qt intrinsic settings for keyboard shortcuts, so it probably does the translation for the app. What is the common keyboard shortcut for replace on a mac then?

I do have a logical flag for Windows, Linux and Darwin, so I can easily wrap the assignment of a keyboard shortcut in an if statement for this one.

**Comment by johnblommers** [comment]
The short answer is that the replace key on macOS should be <kbd>⌘</kbd><kbd>=</kbd>

TL;DR

What is the common keyboard shortcut for replace on macOS? This question actually required me to take a look because it's not a common paradigm in my bubble.

**TextEdit** is probably typical. Here are the edit keyboard shortcuts it offers:  
<kbd>⌘</kbd><kbd>F</kbd> Find ...  
<kbd>⌥</kbd><kbd>⌘</kbd><kbd>F</kbd> Find and Replace...  
<kbd>⌘</kbd><kbd>G</kbd> Find Next  
 <kbd>⇧</kbd><kbd>⌘</kbd><kbd>G</kbd> Find Previous  

**novelWriter**'s edit menu keyboard shortcuts on macOS are:  
<kbd>⌘</kbd><kbd>F</kbd> Find  
none for Replace  
<kbd>F3</kbd> Find Next  
<kbd>⇧</kbd><kbd>F3</kbd> Find Previous  
<kbd>⇧</kbd><kbd>⌘</kbd><kbd>1</kbd> Replace Next  

Of course macOS has special names for some of these keys:  
<kbd>⌘</kbd> Command
<kbd>⇧</kbd> Shift
<kbd>⌥</kbd> Option, the same physical key as <kbd>Alt</kbd>

Saving the best for last, the famous BBEdit programmer's editor is probably our best example. Rather than develop blindness and carpel tunnel syndrome let me post a screenshot of the BBEdit edit menu:

![BBEdit](https://user-images.githubusercontent.com/8484427/68168105-8100ef00-ff1c-11e9-81bc-9920fd60b3f1.png)

**Comment by vkbo** [comment]
Thanks for the feedback. I'll add the shortcut for find next and find previous under an `if darwin` statement. As for the replace and find next, I don't think there's a standard anywhere. I used the key combination from VSCode.

Checking with my office mate, the command = shortcut in all applications we tested changed font size, so not sure that is the correct one.

**Comment by vkbo** [comment]
@johnblommers, I think this issue can be closed now?

**Comment by vkbo** [comment]
Since a few changes have been made here, and I'm not aware of any further issues, I'm closing this.
